documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -329,11 +329,11 @@ NOQuuuWuFMMMCCCCPPPPTTTTTTTTRRRR
 
 ^1^ Information on xref:raspberry-pi.adoc#otp-register-and-bit-definitions[programming the OTP bits].
 
-^2^ Warranty bit is never set on Pi4.
+^2^ Warranty bit is never set on Raspberry Pi 4.
 
-=== New-style revision Codes In Use
+=== New-style Revision Codes in Use
 
-This list is not exhaustive, there may be codes in use in the field that are not in this table.
+This list is not exhaustive: there may be codes in use that are not in this table.
```

Comment:
I wonder if it's worth making this a `NOTE:` ? :shrug:

Reply:
Yes. It should be a `NOTE`. Good call.

Reply:
I've added `NOTE:` to the front of this paragraph, and changed the colon after `exhaustive` to a dash as I think this looks better, rather than two colons. I'm assuming the `NOTE:` gets automagically turned into a box on the website, but this handles the case where someone uses the page content without that feature.